environment.yml

```diff
@@ -10,5 +10,6 @@ dependencies:
 - mantid-workbench
 - periodictable
 - psutil
+- pystog
```

Comment:
@marshallmcdonnell Thanks a lot for the update! I will wait for the CI to finish its job but I guess it will still complain about the pyqt version issue which we may want to ignore the moment?

Reply:
Yep, exactly.
I hate to ignore it but it is a non-trivial task to fix it.

Reply:
Yep, I understand that.